src/test/java/net/datafaker/integration/FakerIntegrationTest.java

```diff
@@ -205,6 +205,7 @@ void testAllFakerMethodsThatReturnStrings(Locale locale, Random random) throws E
         testAllMethodsThatReturnStringsActuallyReturnStrings(faker.yoda());
         testAllMethodsThatReturnStringsActuallyReturnStrings(faker.zelda());
         testAllMethodsThatReturnStringsActuallyReturnStrings(faker.oscarMovie());
+        testAllMethodsThatReturnStringsActuallyReturnStrings(faker.massEffect());
```

Comment:
Please, follow order here too

Reply:
have also moved oscarMovie to the right place.